policyengine_us/variables/gov/states/co/ccap/co_ccap_parent_fee.py

```diff
@@ -0,0 +1,86 @@
+from policyengine_us.model_api import *
+
+
+class co_ccap_parent_fee(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Colorado Child Care Assistance Program parent fee"
+    reference = (
+        "https://docs.google.com/spreadsheets/d/1EEc3z8Iwu_KRTlBtd2NssDDEx_FITqVq/edit#gid=468321263",
+        "https://docs.google.com/spreadsheets/d/1HtPiC2qxclzWfBa7LRo2Uohrg-RCBkyZ/edit#gid=582762342",
+        "https://www.sos.state.co.us/CCR/GenerateRulePdf.do?ruleVersionId=11042&fileName=8%20CCR%201403-1#page=41",
+        "https://www.sos.state.co.us/CCR/GenerateRulePdf.do?ruleVersionId=11042&fileName=8%20CCR%201403-1#page=62",
+    )
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.CO
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.co.ccap
+        person = tax_unit.members
+        # Identify whether the filers are during the entry or re-determination process.
+        is_entry = tax_unit("co_ccap_is_entry", period)
+        entry_eligible = tax_unit("co_ccap_entry_eligible", period)
+        is_re_determination = tax_unit("co_ccap_is_re_determination", period)
+        re_determination_eligible = tax_unit(
+            "co_ccap_re_determination_eligible", period
+        )
+
+        if (is_entry * entry_eligible) | (
+            is_re_determination * re_determination_eligible
+        ):
+            # Calculate base parent fee and add on parent fee.
+            agi = tax_unit("adjusted_gross_income", period)
+            hhs_fpg = tax_unit("tax_unit_fpg", period)
+            num_child_age_eligible = tax_unit(
+                "co_ccap_num_child_age_eligible", period
+            )
+
+            # The numebrs below are weights copied from government spreadsheet (url: )
+            base_parent_fee = np.round(
+                where(
+                    agi <= hhs_fpg,
+                    agi * 0.01 / 12,
```

Comment:
make the 0.01 parameters 
0.01 - base_fee / first_multiplication_factor.yaml

-> in the reference just mention the excel spreadsheet and that the factors are included in the calculation with documentation in the forms